src/MagnumPlugins/UfbxImporter/UfbxImporter.h

```diff
@@ -0,0 +1,354 @@
+    a single set of attributes that are imported, see @ref Trade-UfbxImporter-pbr-attributes
+    for an exhaustive listing.
+-   @ref MaterialAttribute::DiffuseTextureMatrix and similar matrix attributes
+    for other textures are imported.
+-   @ref MaterialAttribute::DiffuseTextureCoordinates and similar UV layer
+    attributes are not supported.
+
+@subsection Trade-UfbxImporter-behavior-lights Light import
+
+-   @ref LightData::Type::Directional and @ref LightData::Type::Ambient expect
+    the attenuation to be constant, but FBX is not required to follow that.
+    In that case the attenuation value from the file is ignored.
+-   Area and volume lights are not supported
+
+@subsection Trade-UfbxImporter-behavior-meshes Mesh import
+
+-   Only triangle meshes are loaded (quad and poly meshes are triangularized)
+-   Vertex creases and any edge or face attributes are not imported
+-   The importer follows types used by ufbx truncated to 32-bit floats, thus indices are always
+    @ref MeshIndexType::UnsignedInt, positions, normals, tangents and
+    bitangents are always imported as @ref VertexFormat::Vector3, texture
+    coordinates as @ref VertexFormat::Vector2 and colors as
+    @ref VertexFormat::Vector4. Most FBX files contain geometry data natively
+    as double-precision floats.
+-   If a mesh contains multiple materials it is split into parts and the node
+    contains each part as a separte mesh/material entry.
+
+The meshes are indexed by default unless cb{.ini} generateIndices @ce @ref Trade-UfbxImporter-configuration "configuration option"
+is disabled. Vertex position is always defined, normals can be missing unless
+cb{.ini} generateMissingNormals @ce @ref Trade-UfbxImporter-configuration "configuration option"
+is set. There are an arbitrary amount of UV/tangent/bitangent/color sets, you
+can specify a maximum limit using cb{.ini} maxUvSets @ce,
+cb{.ini} maxTangentSets @ce, cb{.ini} maxColorSets @ce @ref Trade-UfbxImporter-configuration "configuration options",
+note that setting any to zero disables loading any tangents etc.
+
+@section Trade-UfbxImporter-configuration Plugin-specific configuration
+
+It's possible to tune various import options through @ref configuration(). See
+below for all options and their default values:
+
+@snippet MagnumPlugins/UfbxImporter/UfbxImporter.conf configuration_
+
+See @ref plugins-configuration for more information and an example showing how
+to edit the configuration values.
+
+@section Trade-UfbxImporter-pbr-attributes PBR material attributes
+
+ufbx normalizes PBR materials into a superset of all vendor-specific PBR material
+models that ufbx supports, these attributes are exposed as custom @ref MaterialAttribute
+names.
+
+Majority of the attributes match
+[Autodesk Standard Surface](https://autodesk.github.io/standard-surface/)
+parameters:
+
+Layer | Attribute | Type | OSL parameter
+----- | --------- | ---- | -------------
+Base | baseFactor [1] | Float | base
```

Comment:
Is there a way to make proper footnotes in Doxygen+m.css? I saw some in `.rst` files but the same syntax (unsurprisingly) didn't work here.

Reply:
AFAIK only manually :sweat_smile: I'll look into that locally post-merge.